File: bbarena-model/src/main/java/net/sf/bbarena/model/event/RollModifier.java
package net.sf.bbarena.model.event;

import java.io.Serializable;

public class RollModifier implements Serializable {

	private static final long serialVersionUID = -4692545546332381896L;

	private final int _modifier;
	private final String _description;

	public RollModifier(int modifier, String description) {
		_modifier = modifier;
		_description = description;
	}

	public String getDescription() {
		return _description;
	}

	public int getModifier() {
		return _modifier;
	}

	@Override
	public String toString() {
		return new StringBuilder(_modifier).append(" (").append(_description).append(")").toString();
	}

}

File: src/main/kotlin/bbarena/model/event/RollModifier.kt
package bbarena.model.event

import java.io.Serializable

class RollModifier(val modifier: Int, val description: String) : Serializable {

    override fun toString(): String {
        return StringBuilder().append(modifier).append(" ").append(description).toString()
    }

    companion object {

        private const val serialVersionUID = -4692545546332381896L
    }

}
